doc/snippets/Microsoft.Data.SqlClient/SqlAuthenticationInitializer.xml

```diff
@@ -2,17 +2,17 @@
   <members name="SqlAuthenticationInitializer">
     <SqlAuthenticationInitializer>
       <summary>
-        Called from constructors in derived classes to initialize the <see cref="T:Microsoft.Data.SqlClient.SqlAuthenticationInitializer" /> class.
+        Called during instantiation of the first <see cref="T:Microsoft.Data.SqlClient.SqlConnection" /> instance.
```

Comment:
This actually changed recently (6.0 preview 1) to only be called during instantiation of the first connection that uses the Authentication option (allows chisel to eliminate MSAL deps). See #2636 (netcore) and #2873 (netfx).

Reply:
Thanks, I'd not noticed the first PR. I've updated the documentation to reference that (in both places)